service/service.go

```diff
@@ -141,6 +146,39 @@ func New(ctx context.Context, set Settings, cfg Config) (*Service, error) {
 	return srv, nil
 }
 
+func Validate(ctx context.Context, set Settings, cfg Config) error {
+	tel, err := telemetry.New(ctx, telemetry.Settings{ZapOptions: set.LoggingOptions}, cfg.Telemetry)
+	if err != nil {
+		return fmt.Errorf("failed to get logger: %w", err)
+	}
+
+	telSettings := servicetelemetry.TelemetrySettings{
+		Logger:         tel.Logger(),
+		TracerProvider: tel.TracerProvider(),
+		MeterProvider:  noop.NewMeterProvider(),
+	}
```

Comment:
Why not using `noop` everywhere?

Reply:
Do you mean for the `Logger` and `TracerProvider` fields?  I'm not seeing anything in the `noop` package that could be used to initialize those fields.